LondonBritishLibrary/orient/BLorient594.xml

```diff
@@ -0,0 +1,466 @@
+                                        <width>9.125</width>
+                                    </dimensions>
+                                </extent>
+
+                            </supportDesc>
+
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="20 21"/>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>The manuscript is written in fine hand, of the <date>17th century.</date></desc>
+                              <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
+                            </handNote>
+
+                        </handDesc>
+                        <decoDesc>
+                          <summary>Pictures.</summary>
+                            <decoNote xml:id="d1" type="drawing">
+                              <locus target="#177v"></locus>
+                              <desc>The blessed Virgin and child.</desc>
+                              </decoNote>
+
+                            <decoNote xml:id="d2" type="drawing">
+                              <locus target="#141v"></locus>
+                              <desc>The crucifixion.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                          <list>
+
+                            <item xml:id="a1">
+                              <locus target="#3r"></locus>
+                              <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                              <q xml:lang="gez">አርጋኖን፡ ዘ<placeName ref="INS0101MadhaneAlam">
+                                ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q></item>
+
+                                <item xml:id="a2">
+                                <locus target="#197r"/>
+                                <desc type="GuestText"><title ref="LIT3984basamay">Hymn to the blessed Virgin Mary</title></desc>
+                                <q xml:lang="gez"><seg part="I">በሰማይ፡ ወምድር፡ አልብየ፡ ባዕደ፡ አበ፡ ወእመ፡ እኅተ፡ ወውሉደ፡ ማርያም፡ ድንግል፡ እትአመነኪ፡ ገሃደ፡</seg></q>
+                                </item>
+
+<item xml:id="a3">
+<locus target="#1r"/>
+<desc>Significations of the letters of the alphabet.</desc>
+<q xml:lang="gez">ፊደል፡ ሀ፡ ብሂል፡ ሀልዎቱ፡ ለአም፡ እምቅድመ፡ ዓለም።</q>
+</item>
+
+<item xml:id="a4">
+<locus target="#2"/>
+<desc>On <locus target="#2">f. 2</locus> is a part of a hymn, in older hand. In a later hand, notes on the efficacy of certain prayers.</desc>
+<q xml:lang="gez">ጸሎተ፡ ዘይትገበር፡ በቤተ፡ ክርስቲያን፡ ለአቡነ፡ ዘበሰማያት፡ ፍሬሁ፡ ፴፼ከ፲፻፡ በ፩መዐዝነ፡ ዓለም፡ ይክወን፡ አምሳለ፡ ደመና፡</q>
+</item>
+
+
+<item xml:id="e1">
+<locus target="#198"/>
+<desc><q xml:lang="am"/>Notes in Amharic.</desc>
+</item>
+
```

Comment:
I think this is an addition as well. q should not be within the desc, but after it (see above).